app/controllers/grades_controller.rb

```
@@ -245,6 +264,32 @@ def set_participant_and_team_via_assignment
+    # Export one row per assignment participant. A participant-specific grade
+    # overrides the team submission grade; submission comments remain team-level.
+    def grades_csv_for(assignment, include_email: false)
+        headers = GRADES_EXPORT_HEADERS + (include_email ? GRADES_EXPORT_OPTIONAL_HEADERS : [])
+
+        CSV.generate(headers: true) do |csv|
+            csv << headers
+
+            assignment.participants.includes(:user).find_each do |participant|
+                team = participant.team
+                row = [
+                    participant.user_name,
```

RuboCop (1.86.1)

[convention] Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.

(Layout/FirstArrayElementIndentation)